guides/release/contributing/index.md

```diff
@@ -1,163 +1,35 @@
+# Types of contributions
 
-Security fixes should not introduce new APIs, but may, if strictly
-necessary, break existing APIs. Such breakages should be as limited as
-possible.
+Any change that aims to make an improvement is very welcome!
 
-### Bug Fixes
+You can create issues to document many things (list is not exhaustive!):
 
-#### Urgent Bug Fixes
+- Bugs
+- Ideas for enhancements
+- Code quality (e.g. refactoring)
+- Improving the documentation (clarifying, rephrasing, providing more examples, fixing typos, adding missing details)
 
-Urgent bugfixes are bugfixes that need to be applied to the existing
-release branch. If possible, they should be made on master and prefixed
-with `[BUGFIX release]`.
+Creating an issue is a great way to start a discussion and gather opinions of other members of the Ember community. Once a decision has been made, you or someone else can volunteer to work on it, and create a pull request with their work.
 
-#### Beta Bug Fixes
+Apart from creating new issues and pull request, another way to contribute is comment on existing issues and pull requests. The more eyes we get on something, the lower the chance we'll overlook potential problems.
 
-Beta bugfixes are bugfixes that need to be applied to the beta branch.
-If possible, they should be made on master and tagged with `[BUGFIX
-beta]`.
+# Where to get started
 
-#### Security Fixes
+A great place to start is helping improve something you personally ran into. E.g. if you found some of the documentation unclear yourself, please create an issue to highlight it, and optionally suggest how to simplify it.
 
-Security fixes need to be applied to the beta branch, the current
-release branch, and the previous tag. If possible, they should be made
-on master and tagged with `[SECURITY]`.
+Have a look at the [list of main repositories](https://guides.emberjs.com/release/contributing/repositories/) to learn about the different components of the Ember project.
 
-### Features
+If you need some inspiration, you can check out the [Help Wanted](https://help-wanted.emberjs.com/) dashboard to browse for issues. If you are a beginner, look out for issues with the "Help wanted" and "Good first issue" labels.
 
-Features must always be wrapped in a feature flag. Tests for the feature
-must also be wrapped in a feature flag.
+# Asking for help
 
-Because the build-tools will process feature-flags, flags must use
-precisely this format. We are choosing conditionals rather than a block
-form because functions change the surrounding scope and may introduce
-problems with early return.
+Please comment directly on issues and PRs if you need help. This way others will see and can chime in to help.
+You can also visit emberjs.com/community and join the [Ember.js discord server](https://discord.gg/emberjs). Channels that start with `dev` are for contributors working on the corresponding projects, and they are a great place for questions.
```

Comment:
```suggestion
You can also visit the [Ember.js Community Page](https://emberjs.com/community) to join the Ember.js Discord server. Channels that start with `dev` are for contributors working on the corresponding projects, and they are a great place for questions.
```

I suggest taking out the link to the Discord server so that if it changes in the future, we only have to update the community page.

Reply:
Makes sense, good idea!